languagetool-language-modules/nl/src/main/java/org/languagetool/rules/nl/CompoundAcceptor.java

```diff
@@ -558,14 +558,14 @@ public class CompoundAcceptor {
     }
   }
 
-  private final Tagger tagger;
+  private final DutchTagger DutchTagger;
```

Comment:
The class is capitalized (`DuchTagger`), but the variable should be lowercase (`dutchTagger`).